ucm2/codecs/tfa9895/Speaker.conf

```diff
@@ -0,0 +1,9 @@
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+
+	Value {
+		PlaybackChannels 1
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId},2"
```

Comment:
This line is specific to the msm8916 devices and not specific to tfa9895. So while it's a bit annoying I would prefer if you just duplicate the `SectionDevice."Speaker"` part to both samsung-a2015/HiFi.conf and samsung-gmax/HiFi.conf. Otherwise it's a bit confusing.

Reply:
Note that I merged your other PR, so please also use PlaybackChannels 1 for gmax.